components/ButtonAvatar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { TouchableOpacity, Image } from 'react-native'
+import { styles } from './ButtonAvatar.styles'
+
+export const ButtonAvatar = (props) => {
+
```

Comment:
we could make this component responsible for if there is an image or not.
So if not, it will show a blue circle instead?

I know we want to use a default image, but it would be more robust to have a "fallback" here?

Reply:
Yes, the fallback is something, I wanted to tackle now. So, will put that into the next PR depending on this. And the other two comments, I will also add there! So, will merge now. :partying_face:

Reply:
Actually, it already does that. When there is no image in the props (e.g. if the image is null), the blue circle is shown. Should we make that more robust and more explicit? What would be the best way? @khamui

Reply:
ah okay, so we could shift that logic to this location. so in case we use the AvatarCircle somewhere else it handles that and we do not need to think about the blue circle fallback again? we can have a look together as well @fuzzylogic2000 ?